src/calendar-list/index.js

```diff
@@ -44,6 +44,10 @@ class CalendarList extends Component {
     calendarWidth: PropTypes.number,
     /** Dynamic calendar height */
     calendarHeight: PropTypes.number,
+
+    // Should Keyboard persist taps
+    keyboardShouldPersistTaps: PropTypes.oneOf(['never', 'always', 'handled']),
```

Comment:
defining the propType as `FlatList.propTypes. keyboardShouldPersistTaps` is clearer.